api/core/api.proto

```diff
@@ -97,14 +99,16 @@ message ListenResultRequest {
 //   "executionID": "__EXECUTION_ID__",
 //   "taskKey":     "__TASK_KEY__",
 //   "outputKey":   "__OUTPUT_KEY__",
-//   "outputData":  "{\"foo\":\"bar\"}"
+//   "outputData":  "{\"foo\":\"bar\"}",
+//   "tags":        ["executionX", "test"]
 // }
 // ```
 message ResultData {
-  string executionID = 1; // The unique identifier of the execution.
-  string taskKey = 2;     // The key of the executed task.
-  string outputKey = 3;   // The output's key from the returned task.
-  string outputData = 4;  // The output's data from the returned task, encoded in JSON.
+  string executionID = 1;   // The unique identifier of the execution.
+  string taskKey = 2;       // The key of the executed task.
+  string outputKey = 3;     // The output's key from the returned task.
+  string outputData = 4;    // The output's data from the returned task, encoded in JSON.
+  repeated string tags = 5; // The list of tags associated to the execution
```

Comment:
Can you add "The" at the beginning of the comment to look the same as other comments